src/actions/websocket.js

```diff
@@ -671,6 +682,25 @@ function handleAddEmoji(msg, dispatch) {
     });
 }
 
+function handleLicenseChangedEvent(msg, dispatch) {
+    const data = msg.data.license;
+
+    dispatch({
```

Comment:
what happens when the config does not change but the license does? same as above look at https://github.com/mattermost/mattermost-mobile/blob/master/app/mattermost.js#L82

Reply:
See the linked PR above, but basically the `SERVER_VERSION_CHANGED` implicitly includes changes to the config and license given the need to support older servers. Either way, we will receive discrete websocket events for the license changing even if the config does not change.